metadata-models/src/main/pegasus/com/linkedin/common/GlobalTags.pdl

```diff
@@ -0,0 +1,12 @@
+namespace com.linkedin.common
+
+/**
+ * Tag aspect used for applying tags to an entity
+ */
+record GlobalTags {
```

Comment:
Why is `Global` in here? Is there ever going to be some `Local` tag?

Reply:
The legacy tags would be the `local` tags here- the intention was to create a name that wouldn't collide with the existing freeform string tags